src/main/java/com/deavensoft/paketomat/center/model/Paketomat.java

```diff
@@ -2,17 +2,14 @@
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.*;
-
 import javax.persistence.*;
 import java.util.List;
-
-
 @Getter
 @Setter
 @AllArgsConstructor
 @NoArgsConstructor
 @Entity
-@Table(name = "paketomat")
+@Table(name = "paketomat", schema = "public")
```

Comment:
Why was schema = "public" added?

Reply:
Exceptions are thrown when we don't have that part

Reply:
Try to resolve this problem adding spring.jpa.properties.hibernate.default_schema=schema-name in the configuration file.

Reply:
Resolved